spec/integration/_dev/benchmark/spec.yml

```diff
@@ -6,12 +6,32 @@ spec:
       type: folder
       name: system
       required: false
-      additionalContents: false
+      additionalContents: true
       contents:
         - description: Folder containing deployable services for benchmarks.
           type: folder
           name: deploy
-          $ref: "./deploy/spec.yml"
+          additionalContents: false
+          required: false
+          contents:
+            - description: Folder containing deployment files for a Docker-based service deployment
+              type: folder
+              name: docker
+              $ref: "./deploy/docker/spec.yml"
+            - description: Folder containing deployment files for a Terraform-based service deployment
+              type: folder
+              name: tf
+              $ref: "./deploy/tf/spec.yml"
+            - description: Folder containing deployment files for a Kubernetes service deployment
+              type: folder
+              name: k8s
+              $ref: "./deploy/k8s/spec.yml"
+            - description: Configuration file describing variations available for deployment
+              type: file
+              name: "variants.yml"
+              contentMediaType: "application/x-yaml"
+              required: false
+              $ref: "./deploy/variants.spec.yml"
```

Comment:
Same for this `variants.yml`. Could it be set the `$ref` key to be https://github.com/elastic/package-spec/blob/main/spec/integration/_dev/deploy/variants.spec.yml ?